setup.py

```diff
@@ -29,10 +29,12 @@
         'pyhocon>=0.3.53',
         'requests',
         'pyopenssl',
-        'autouri>=0.2.0',
+        'autouri>=0.2.1',
         'miniwdl',
         'humanfriendly',
         'numpy',
         'pandas',
+        'scikit-learn',
+        'matplotlib',
```

Comment:
Should add version specification. Exactly pinning is ideal but at the least should specify a major version

Reply:
Done.